settings.py

```diff
@@ -4,7 +4,7 @@
 import os
 root_dir = os.path.dirname(os.path.abspath(__file__))
 
-DEBUG = False
+DEBUG = "Development" in os.environ.get("SERVER_SOFTWARE", "")
```

Comment:
Can this be documented somewhere? Also usually the environment variable would look something like `DEBUG=1`.